breakpad-symbols/src/sym_file/types.rs

```diff
@@ -33,6 +34,12 @@ pub struct SourceLine {
     pub line: u32,
 }
 
+impl SourceLine {
+    pub fn end_address(&self) -> u64 {
+        self.address.saturating_add(self.size.into())
+    }
+}
```

Comment:
hmm saturating always feels like a trap, why do you think this is better than discarding the entry?

Reply:
I don't have strong opinions on this. I used saturating simply because then I don't have to deal with None.
We may produce garbage in the bad cases but at least we won't panic.

Reply:
My default calculus here is that one Bad input can spoil the batch if we keep it around, but discarding a sketchy input (as in discarding the line) is unlikely to even be noticed, since we have millions of lines and you're gonna query like 20 of them. The places where I go out of my way to preserve sketchy inputs are places where it was an actual problem in production.

Reply:
Filtering them out sounds good to me. But we use the same type for unfiltered and filtered line records. Should we return Option from end_address or not?